tests/Functional/SprykerEco/Zed/ArvatoRss/Business/Facade/ArvatoRssFacadeRiskCheckTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Unit\SprykerEco\Zed\ArvatoRss\Business\Api\Mapper;
+
+use PHPUnit\Framework\TestCase;
+use SprykerEco\Zed\ArvatoRss\Business\ArvatoRssFacade;
+
+class RiskCheckRequestConverterTest extends TestCase
+{
+
+    public function performRiskCheckTest()
+    {
+        $facade = new ArvatoRssFacade();
```

Comment:
Too many empty lines.

Reply:
If CS would be run, it would also find 50+ other issues and auto-fix them.